### PR TITLE
fix no-index error

### PR DESCRIPTION
**Pull request by mirche97**
I ran into an issue using php7.4 of not defined index. Quick fix to make it work.

### CODE REVIEW

**Review by robertvanlienden — COMMENTED** [review]
2 comments on the code.

And if possible, please add some useful description to this PR @mirche97.

For the rest; Nice improvements!

File: src/Filters/GeoDistanceFilter.php [hunk 1]
```diff
@@ -24,16 +24,16 @@ public function getElasticaQuery()
...
+
         $this->extend('updateValue', $value);
 
         $mapsKey = self::config()->get('maps_key');
         if (!$mapsKey) {
             throw new \Exception('Maps key is empty');
         }
 
-        if ($value['Search']) {
+        if ($value && $value['Search']) {
```
[the remark this answers is not in the document]
Comment (mirche97) [reply src/Filters/GeoDistanceFilter.php]: Fix this

File: src/Filters/GeoDistanceFilter.php [hunk 2]
```diff
@@ -24,16 +24,16 @@ public function getElasticaQuery()
...
             $data = file_get_contents("https://maps.googleapis.com/maps/api/geocode/json?address=" . urlencode($search) . "&key={$mapsKey}");
```
[the remark this answers is not in the document]
Comment (mirche97) [reply src/Filters/GeoDistanceFilter.php:37]: Not sure if needed, but not relevant atm.

[the remark this answers is not in the document]
Comment (robertvanlienden) [reply src/Filters/GeoDistanceFilter.php:37]: Hmm, for now accept this and let's say: Better double-url-encoded then not-url-encoded?

Only thing i'm now worring about is the `$this->extend('updateValue', $value);`.
Seems like this extension point never did something (eg; The $value is never used after the extension point)?

[the remark this answers is not in the document]
Comment (mirche97) [reply src/Filters/GeoDistanceFilter.php:37]: Not relevant atm.

**Pull request merged**